### ISSUE TITLE
Data Export - Filenames too long for Windows

### ISSUE THREAD

**Issue by ErnieZapata** [dbeaver/dbeaver#19986]
### Description

I previously created a task to export tables in my database as Json files. The task's "Output" was configured as:

![image](https://github.com/dbeaver/dbeaver/assets/20826095/a5535aba-1feb-41dc-8c18-b43a1ccfe2c0)

I would run the task at the start of a test session to capture the table content as Json files and created a directory as described above. There would be 1 Json for each table I selected. I would run my tests and then run the task again to produce a second folder with the updated table content. 

I could then use my file/directory comparison tool to compare and verify updates.

It has been a few releases since I did that, but now, when I execute the task using Community Edition 23.0.4, I get an error: 

```
Task execution failed

Reason:
Data transfer IO error
```
And the details show:
```
Data transfer IO error
  C:\Users\********\AppData\Local\Temp\export-1684263346660-1684263346671-1684263346681-1684263346690-1684263346701-1684263346719-1684263346743-1684263346759-1684263346775-1684263346812-1684263346836-1684263346851-1684263346872-1684263346885-1684263346900-1684263346915-1684263346927-1684263346939-1684263346939.json (The filename, directory name, or volume label syntax is incorrect)
  C:\Users\********\AppData\Local\Temp\export-1684263346660-1684263346671-1684263346681-1684263346690-1684263346701-1684263346719-1684263346743-1684263346759-1684263346775-1684263346812-1684263346836-1684263346851-168426
[logs.zip](https://github.com/dbeaver/dbeaver/files/11491187/logs.zip)
3346872-1684263346885-1684263346900-1684263346915-1684263346927-1684263346939-1684263346939.json (The filename, directory name, or volume label syntax is incorrect)

```
The files created appear as:

![image](https://github.com/dbeaver/dbeaver/assets/20826095/11308881-8ce2-4c93-a9e8-fadb2e07eebe)

There is a single "export.json", then an "export-1684263346660.json", then "export-1684263346660-1684263346671.json", etc and eventually the file name exceeds Windows 256 character limit and bombs.

### DBeaver Version

Community Edition 23.0.4

### Operating System

Windows 10 Professional

### Database and driver

1. Database - MySQL 5.7.33
2. Driver - MySQL Connector Java mysql-connector-java-5.1.44

**Comment by ShadelessFox** [comment]
Hello @ErnieZapata,

Make sure you have "on object data file name conflict" set to "overwrite":

![image](https://github.com/dbeaver/dbeaver/assets/35821147/7a7dae67-cc0e-4df9-9f8d-fb5c28947335)

Please let me know if it solves your issue.

**Comment by ErnieZapata** [comment]
I changed this setting to **Overwrite** but it made no difference, same result.

I noticed that **Write to single file** was disabled, but yet, tasks.json showed:

![image](https://github.com/dbeaver/dbeaver/assets/20826095/9bb1b803-3ce2-4487-bd2b-2730dc21a94c)

I edited the file and changed the value for "useSingleFIle" to false and got the results I want. A new folder gets created with current date/time, and one Json file for each table is created. Running task again, I see a new folder with Json for each table as expected.

Not sure when this **Write to single file** was disabled, but the Json value for this does not seem to reflect an unchecked checkbox. As mentioned before, I created this task with an older version of  DBeaver a while back but just have not used it in a while.

I even delete the old task and recreated and see the same behavior for this setting. I have a work around, so I good for now
